mediaflow_proxy/routes/playlist_builder.py

```
@@ -0,0 +1,388 @@
+    """Scarica una playlist M3U in modo asincrono e restituisce le righe."""
+    headers = {
+        'User-Agent': settings.user_agent,
+        'Accept': '*/*',
+        'Accept-Language': 'en-US,en;q=0.9',
+        'Accept-Encoding': 'gzip, deflate',
+        'Connection': 'keep-alive'
+    }
+    lines = []
+    try:
+        async with httpx.AsyncClient(verify=False, timeout=30) as client:
+            async with client.stream('GET', url, headers=headers) as response:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)